include/libtorrent/torrent_info.hpp

```diff
@@ -590,7 +590,7 @@ TORRENT_VERSION_NAMESPACE_3
 
 		// internal
 		aux::vector<aux::merkle_tree, file_index_t>& internal_merkle_trees();
-		void internal_load_merkle_trees(std::vector<std::vector<sha256_hash>> const& t);
+		void internal_load_merkle_trees(std::vector<std::vector<sha256_hash>> t);
```

Comment:
even though this function doesn't need a copy of this vector (it already copies it unconditionally), it takes it by value just to let the caller give up ownership, because we don't it to hang around. Perhaps this is a bit subtle, but at least there is an assert to ensure it happens